README.md

```diff
@@ -44,7 +48,7 @@ All notable changes to the Pony compiler and standard library will be documented
 ```
 
 ```bash
-changelog-tool release CHANGELOG.md 0.13.1
+changelog-tool release 0.13.1
```

Comment:
This seems like a breaking change to me, do I have to go to every place that `release` is used and change it?

It seems like now I need to use `-f` to set the file.

That would definitely make this a 0.3.0 version and I'll need to update every usage of changelog tool for releasing.

Reply:
Yeah this is 100% a breaking change, I tested it. There are multiple breaking changes.